src/core/server/services/comments/pipeline/phases/tagExpertAnswers.ts

```diff
@@ -24,10 +27,6 @@ export const tagExpertAnswers: IntermediateModerationPhase = ({
           type: GQLTAG.EXPERT,
           createdAt: now,
         },
-        {
-          type: GQLTAG.FEATURED,
-          createdAt: now,
-        },
```

Comment:
Doing this later as we want to only feature if it's a top level comment and we don't have enough comment data during phases to calculate that.

If we only check on parentID, nested comments will be marked as answered via featuring. 
If we try and use ancestorID lists, those aren't populated yet since this "comment" still doesn't exist yet during mod phases.